frontend/awx/administration/notifiers/NotifierForm.tsx

```diff
@@ -209,11 +265,93 @@ function NotifierForm(props: { mode: 'add' | 'edit' }) {
             )}
           </PageFormWatch>
         </PageFormSection>
+        <PageFormSection>
+          <PageFormSwitch
+            labelHelp={
+              <Trans>
+                Use custom messages to change the content of notifications sent when a job starts,
+                succeeds, or fails. Use curly braces to access information about the job: <br />
+                <br />
+                {{ job_friendly_name }}, {{ url }}, {{ status }}.<br />
+                <br />
+                You may apply a number of possible variables in the message. For more information,
+                refer to the
+                <a href="https://docs.ansible.com/automation-controller/latest/html/userguide/notifications.html#create-custom-notifications">
+                  Ansible Controller Documentation.
+                </a>
+              </Trans>
+            }
+            name={'customize_messages'}
+            label={t('Customize messages...')}
+          />
```

Comment:
Customize message switch clears contents of form on toggle

https://github.com/ansible/ansible-ui/assets/112966709/f849b9d7-18f2-4550-a9d1-2118c949dfb6

Reply:
Yeah, I know, I was not sure that this is the problem, but you are right, it should be implemented. The problem is, when PageFormComponents get unmounted, they also seems to delete the data. I will have to insert the data again to the context.

Reply:
If this fix will take a long time, maybe we can close out this PR right now and we can add it as an issue on Jira to be fixed during hardening?

Reply:
I hope this is fixed now :-)

Reply:
Sorry just realized the issue here is fixed